### ISSUE TITLE
Create Vegetation Shader

### ISSUE THREAD

**Issue by Madecu** [carla-simulator/carla#7583]
Unify vegetation Parent Material

Set material instances SubSurface Scaterring and other parameters

**Comment by Madecu** [comment]
We encountered some lighting issues due to the normals of the tree leaves.


![Image](https://github.com/carla-simulator/carla/assets/54811869/62984acf-c258-4f79-bb0d-0dd3e34a38b6)

We resolved it by disabling the TwoSided option in the shader and duplicating the leaf planes, flipping their normals. Using this approach along with TwoSidedSign, we now have normals that behave more realistically, improving the lighting and integration of the trees in the scene.



![Image](https://github.com/carla-simulator/carla/assets/54811869/71aa0e87-f058-4bba-85bf-3e18a72ed246)

(Left; old normals, Right; new normals)
